Create API v1 (#1)
wayscript/wayscript-tinydb: README.md

# TinyDB API Lair template

A simple [TinyDB](https://tinydb.readthedocs.io/en/latest/) API Lair template that can be used to provision databases quickly.

## Setup

To start using this API -

> 1. Create a new Lair with this file as a template

Reviewer: ```suggestion
1. Create a new WayScript Lair from this template
```

> 2. Deploy lair by running the pre-configured deploy trigger

Reviewer 2: Since this is for storage, we should add guidance that they should use this lair via the dev endpoint, so deploys don't overwrite the data

> You can checkout the available API endpoints by visiting the [base endpoint](https://docs.wayscript.com/platform/lairs/endpoints) of your deployed Lair.

Reviewer: We use "default" in the UI.
```suggestion
You can checkout the available API endpoints by visiting the [default endpoint](https://docs.wayscript.com/platform/lairs/endpoints) of your deployed Lair.
```
